packages/components/src/index.ts

```diff
@@ -1,11 +1,11 @@
-export { default as Banner } from "./Banner";
 export type { BannerProps } from "./Banner";
+export { default as Banner } from "./Banner";
 export { default as Button } from "./Button";
 export { default as ClickableStyle } from "./ClickableStyle";
 export { default as Heading } from "./Heading";
-export { default as SvgIcon } from "./SvgIcon";
-export type { IconProps } from "./SvgIcon";
 export { default as Link } from "./Link";
+export { default as SvgIcon } from "./SvgIcon";
 export { default as Tag } from "./Tag";
 export { default as Text } from "./Text";
 export { default as Toast } from "./Toast";
+export type { IconProps } from "./SvgIcon";
```

Comment:
Our plop alphabetizes this file. Even though I didn't keep around any test components, I kept the alphabetization.